pkg/cmd/kind/version/version_test.go

```diff
@@ -60,3 +60,66 @@ func TestTruncate(t *testing.T) {
 		})
 	}
 }
+
+func TestVersion_SuccessWithGitCommitCountAndHash(t *testing.T) {
```

Comment:
If we're going to mutate these globals anyhow, we might as well use fixed inputs and assert the output and just do it in one Test function over each case

Reply:
> If we're going to mutate these globals anyhow, we might as well use fixed inputs and assert the output and just do it in one Test function over each case

I just pushed a commit, is that what you mean?